### PR TITLE
Add links to docs in markdowns under root of repo

### PR DESCRIPTION
**Pull request by vivek-koppuru**
*Issue #, if available:*

*Description of changes:*

Avoid having to maintain to different places where these contents are being stored.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by abhay-krishna** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/aws/eks-anywhere/pull/290#" title="Author self-approved">vivek-koppuru</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=aws%2Feks-anywhere).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/aws/eks-anywhere/blob/main/OWNERS)~~ [vivek-koppuru]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: CODE_OF_CONDUCT.md
Comment (vivek-koppuru) [inline CODE_OF_CONDUCT.md:1]: Should we have these point to the website instead of GitHub?

**Comment by abhay-krishna** [comment]
@vivek-koppuru: The following test **failed**, say `/retest` to rerun all failed tests:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
eks-anywhere-cluster-controller-tooling-presubmit | 5a37363bed871f4774f8b54520e940013058b0f9 | [link](https://prow.eks.amazonaws.com/view/s3/prowpresubmitsdataclusterstack-prowbucket7c73355c-vfwwxd2eb4gp/pr-logs/pull/aws_eks-anywhere/290/eks-anywhere-cluster-controller-tooling-presubmit/1441088055631941632) | `/test eks-anywhere-cluster-controller-tooling-presubmit`

[Full PR test history](https://prow.eks.amazonaws.com/pr-history?org=aws&repo=eks-anywhere&pr=290). [Your PR dashboard](https://prow.eks.amazonaws.com/pr?query=is:pr+state:open+author:vivek-koppuru).

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

File: CODE_OF_CONDUCT.md
Comment (rothgar) [reply CODE_OF_CONDUCT.md:1]: Personally I prefer them to exist in Github. The docs can provide further content but if I have the code checked out locally it's easier to read plain markdown in my IDE

File: docs/content/en/docs/community/contributing.md
Comment (rothgar) [inline docs/content/en/docs/community/contributing.md:40]: Should we make some example discussions in the repo and link to them?

File: docs/content/en/docs/community/contributing.md
Comment (rothgar) [inline docs/content/en/docs/community/contributing.md:58]: We also have "docs" :)

File: docs/content/en/docs/community/contributing.md
Comment (rothgar) [inline docs/content/en/docs/community/contributing.md:73]: @jaxesn are we keeping the roadmap in this repo? If so we should link to it from the main containers-roadmap repo and update the blog link.

File: docs/content/en/docs/community/contributing.md
Comment (rothgar) [inline docs/content/en/docs/community/contributing.md:88]: Is there a benefit of changing this? I've talked about it in the past and prefer "the" because as the community contributes features the word "our" has connotations that this isn't a community project

File: docs/content/en/docs/community/contributing.md
Comment (rothgar) [inline docs/content/en/docs/community/contributing.md:97]: We should switch all of the numbers to markdown style with the same number so items can more easily be reordered if necessary
```
1.
1.
1.
```
will render as 
1.
2.
3.

File: docs/content/en/docs/community/contributing.md
Comment (rothgar) [inline docs/content/en/docs/community/contributing.md:115]: Do we (or should we) have a "good-first-issue" tag like Kubernetes?

File: docs/content/en/docs/community/contributing.md
Comment (vivek-koppuru) [reply docs/content/en/docs/community/contributing.md:40]: I was planning on creating the Discussions structure in the meantime as well based on this review.

File: docs/content/en/docs/community/contributing.md
Comment (vivek-koppuru) [reply docs/content/en/docs/community/contributing.md:73]: Plan is to have milestones immediately, but not opposed to a roadmap

File: docs/content/en/docs/community/contributing.md
Comment (vivek-koppuru) [reply docs/content/en/docs/community/contributing.md:88]: I just missed this

File: docs/content/en/docs/community/contributing.md
Comment (vivek-koppuru) [reply docs/content/en/docs/community/contributing.md:97]: But would it render as such on GitHub markdown?

File: docs/content/en/docs/community/contributing.md
Comment (vivek-koppuru) [reply docs/content/en/docs/community/contributing.md:115]: I believe we do, we just haven't used it yet but I can add it here

File: docs/content/en/docs/community/contributing.md
Comment (vivek-koppuru) [reply docs/content/en/docs/community/contributing.md]: Cool confirmed

File: CODE_OF_CONDUCT.md
Comment (vivek-koppuru) [reply CODE_OF_CONDUCT.md:1]: Yup that's why I linked them this way

**Comment by abhay-krishna** [comment]
@vivek-koppuru: The following tests **failed**, say `/retest` to rerun all failed tests:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
eks-anywhere-cluster-controller-tooling-presubmit | 5a37363bed871f4774f8b54520e940013058b0f9 | [link](https://prow.eks.amazonaws.com/view/s3/prowpresubmitsdataclusterstack-prowbucket7c73355c-vfwwxd2eb4gp/pr-logs/pull/aws_eks-anywhere/290/eks-anywhere-cluster-controller-tooling-presubmit/1441088055631941632) | `/test eks-anywhere-cluster-controller-tooling-presubmit`
eks-anywhere-attribution-files-presubmit | caf0b7957ae019010027f4bd4f5780fb9236ed3c | [link](https://prow.eks.amazonaws.com/view/s3/prowpresubmitsdataclusterstack-prowbucket7c73355c-vfwwxd2eb4gp/pr-logs/pull/aws_eks-anywhere/290/eks-anywhere-attribution-files-presubmit/1441117830643716096) | `/test eks-anywhere-attribution-files-presubmit`

[Full PR test history](https://prow.eks.amazonaws.com/pr-history?org=aws&repo=eks-anywhere&pr=290). [Your PR dashboard](https://prow.eks.amazonaws.com/pr?query=is:pr+state:open+author:vivek-koppuru).

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

**Comment by rothgar** [comment]
Because this has changes outside of `/docs` I can't approve it but it lgtm

**Review by jaxesn — APPROVED** [review]
/lgtm

**Comment by abhay-krishna** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/aws/eks-anywhere/pull/290#pullrequestreview-763540826" title="LGTM">jaxesn</a>*, *<a href="https://github.com/aws/eks-anywhere/pull/290#" title="Author self-approved">vivek-koppuru</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=aws%2Feks-anywhere).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/aws/eks-anywhere/blob/main/OWNERS)~~ [jaxesn,vivek-koppuru]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
